### DISCUSSION TITLE
SFTP split screen for move/copy operations

### DISCUSSION THREAD

**Discussion by r8plicant** [isontheline/pro.webssh.net#1064] [Ideas]
Hi,
it would be great to have a "split screen view" for move and copy operations, allowing us to keep 2 tabs open in 2 panes, it would make copy/move operations in SFTP much faster and smoother. This UI could be triggered by a button or by rotating the screen landscape and should allow us to select which connection to keep in the open tabs and copy/move files around by dragging.
See the attached gif from a similar app to understand better what I'm trying to explain.

This could be related to issue #57.
Thanks

![copy-files](https://github.com/isontheline/pro.webssh.net/assets/76573019/41e35802-b9ea-47fc-bd15-0baa90989e24)

**Comment by isontheline** [comment]
Hi @r8plicant 👋

Many thanks for your request that make senses 🙏

It could be related to #57 but I prefer to have distinct issues for this case => I'm opening an issue right now : #1065
